myInstagram/myInstagram/Controller/SearchTableViewController.swift

```diff
@@ -0,0 +1,177 @@
+//
+//  SearchTableViewController.swift
+//  myInstagram
+//
+//  Created by Илья on 23.10.2022.
+//
+
+import UIKit
+
+/// Страница поиска
+final class SearchTableViewController: UITableViewController {
+    
+    // MARK: - Constants
+    enum Constans {
+        enum AvatarImageNames {
+            static let ilentiy = "MyPhoto"
+            static let ledenev = "ledenevAvatar"
+            static let mem = "memAvatar"
+            static let ledenevContent = "ledenevContent1"
+        }
+        
+        enum Identifiers {
+            static let profile = "Profile"
+            static let detail = "Detail"
+            static let hightlights = "Highlights"
+            static let posts = "Posts"
+        }
+        
+        enum NickNames {
+            static let ilentiy = "ilentiy"
+            static let ledenev = "alex_ledenev"
+            static let mem = "memolog"
+        }
+    }
+    
+    enum UserInfo {
+        enum Ilentiy {
+            static let nickname = "ilentiy"
+            static let avatarImageName = "MyPhoto"
+            static let posts = "15"
+            static let subscribers = "124"
+            static let subscriptions = "45"
+            static let alias = "Шо то там откуда то там"
+            static let activity = "Предприниматель"
+            static let description = """
+            На небе только и разговоров, что о море и о закате.
+            Там говорят о том, как чертовски здорово наблюдать
+            за огромным огненным шаром, как он тает в волнах.
+            """
+        }
+    }
+    
+    private enum TableCellTypes {
+        case profile
+        case detail
+        case hightlights
+        case posts
+    }
+    
+    // MARK: - Visual Component
+    private let refresherControl = UIRefreshControl()
```

Comment:
```suggestion
    private let refreshControl = UIRefreshControl()
```
er Илюша ты в порядке?)